src/App.js

```diff
@@ -1,25 +1,151 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+import Boxes from "./Components/Boxes";
```

Comment:
This is me being super picky but it would probably get picked up in a big business code review

Make sure all components are imported on the same lines (Makes its easier to delete if they arent used anymore because they are then effectively grouped together (Just move the app.css import either above the boxes or below the colourpicker